### ISSUE TITLE
UX Suggestion: Add a next/prev button at the end of each doc page

### ISSUE THREAD

**Issue by hawkticehurst** [microsoft/vscode-docs#5432]
While reading the docs in sequential order, I've often found it to be disrupting/disorienting when I get to the end of a doc page and then am required to mouse over to the sidebar, remember the title of the page I was reading, and then find and click the next doc in sequence.

I would love it if "next" and "previous" buttons were added to the end of each doc page to ensure a smoother experience while reading the docs in order.

cc @daviddossett

**Comment by daviddossett** [comment]
@gregvanl curious to hear your thoughts here.

**Comment by gregvanl** [comment]
I'm not so sure about adding this UI. We have a "Next steps" section for some topics and not every topic has a logical next or previous topic. 
We can leave this open and see if folks in the community upvote. I've never heard of this ask before.
Could be an option for certain topics, such as the UX Guidelines, where I could imagine someone might want to work their way through the entire set?

**Comment by hawkticehurst** [comment]
Oh yeah to clarify I was thinking of this is as less about adding contextualized next steps for the content but is more of generalized UI/UX mechanism for the purpose of navigating the docs. It just gives readers another option for navigating that (in my experience) feels a bit easier than searching through the sidebar for the next doc page.

I also see this bit of UI used a ton in other doc websites and is where I got the idea from. Here are a couple of examples if it helps paint a better picture of what I mean (scroll to bottom of pages):

- [Vite example](https://vitejs.dev/guide/#community)
- [React example](https://beta.reactjs.org/learn/thinking-in-react#where-to-go-from-here)
- [Next.js example](https://nextjs.org/docs/basic-features/pages#learn-more)
- [Vue example](https://vuejs.org/guide/essentials/application.html#multiple-application-instances)

At the same time, a hybrid approach where related content (like UX guidelines or extension capabilities) have next/prev links and one-off content like the extension guides don't have them could be a nice in-between too! I think I've seen some doc sites take that approach before.

**Comment by KarlGusta** [comment]
Yeah. I have interacted with this too, and I think it is a good idea. Docker's documentation uses this too and also Chakra UI.

**Comment by stech11845** [comment]
May be nice for some long doc pages, but not short ones.

**Comment by hawkticehurst** [comment]
> May be nice for some long doc pages, but not short ones.

I worry this would lead to more confusion than it would help if there were inconsistencies in UI from doc page to doc page.
